CONTRIBUTING.md

```diff
@@ -51,6 +51,79 @@ For source code integrity , Hyperledger Bevel GitHub pull requests are accepted
 8. Work on the feedbacks to revise the PR if there are any comments
 9. If not, the PR gets approved , delete feature branch post the merge
 
+## Create local branch
+
+> Whenever you begin work on a new feature or bugfix, it's important that you create a new branch.
+1. Clone your fork to your local machine
+
+2. Setup your local fork to keep up-to-date (optional)
+   ```
+   # Add 'upstream' repo to list of remotes
+   git remote add upstream https://github.com/hyperledger/bevel.git
+   # Verify the new remote named 'upstream'
+   git remote -v
+   # Checkout your main branch and rebase to upstream.
+   # Run those commands whenever you want to synchronize with main branch
+   git fetch upstream
+   git checkout main
+   git rebase upstream/main
+   ```
+3. Create your branch.
+   ```
+   # Checkout the main branch - you want your new branch to come from main
+   git checkout main
+   # Create a new branch named `<newfeature>` (give simple informative name)
+   git branch <newfeature>
```

Comment:
I would rather change it to
`git branch feat-issue_number` 
or
`git branch bug-issue_number`
or 
`git branch docs-issue_number`

In this way, we can create PR and commit titles accordingly in the future. For example:
If the issue is a feature for fabric, 
The branch can be `feat-000`
And then the PR title can be `[fabric] feat: added support for sample_feature`
And then the commit can be something like
```
feat(fabric): added support for sample_feature  
   Updated ....  
   Fixed .....  
Fixes #000
 ```